File: abc114b.cs
using System;

public class ABC114B {
  static public void Main(string[] args){
    string[] buf;
    buf = Console.ReadLine().Split(' ');
    string S = buf[0];
    int target = 753;
    int minDiff = int.MaxValue;
    for (int i = 0; i < S.Length - 2; i++) {
      int num = int.Parse(S.Substring(i, 3));
      int diff = Math.Abs(num - target);
      if (diff < minDiff) {
        minDiff = diff;
      }
    }
    Console.WriteLine(minDiff);
  }
}

File: abc114b.py

def main():
    buf = input()
    S = buf
    target = 753
    min_diff = 999
    for i in range(len(S) - 2):
        num = int(S[i:i+3])
        diff = abs(num - target)
        if diff < min_diff:
            min_diff = diff
    print(min_diff)


if __name__ == '__main__':
    main()
